manifests/j/Jabra/Direct/5.5.39795/Jabra.Direct.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Jabra.Direct
+PackageVersion: 5.5.39795
+MinimumOSVersion: 7.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /Install /Quiet /Norestart
+  SilentWithProgress: /Install /Passive /quiet
+  Interactive: /Install
+ProductCode: '{2183fe12-ffa1-4abe-a5a2-acffcee16462}'
+Installers:
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://jabraxpressonlineprdstor.blob.core.windows.net/jdo/JabraDirectSetup.exe
+  InstallerSha256: 3BFCA39F2E794C097BA10F3BAF8AEC02BD4F3119D2A1524BAED9944888CC0374
```

```suggestion
Installers:
- Architecture: x86
  InstallerType: exe
  Scope: machine
  InstallerUrl: https://jabraxpressonlineprdstor.blob.core.windows.net/jdo/JabraDirectSetup.exe
  InstallerSha256: 3BFCA39F2E794C097BA10F3BAF8AEC02BD4F3119D2A1524BAED9944888CC0374
  ProductCode: '{2183fe12-ffa1-4abe-a5a2-acffcee16462}'
  InstallerSwitches:
    Silent: /Install /Quiet /Norestart
    SilentWithProgress: /Install /Passive /quiet
    Interactive: /Install
```
Just a cleanup because it confuses me